src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -261,6 +261,7 @@ void startStop() throws IOException, InterruptedException {
 
     // `terra notebook start --name=$name`
     TestCommand.runCommandExpectSuccessWithRetries("notebook", "start", "--name=" + name);
-    GcpNotebookUtils.assertNotebookState(name, "ACTIVE");
+    GcpNotebookUtils.assertNotebookState(name, "PROVISIONING");
```

Comment:
Do we need to assert this? 
Reason for ask - if the notebook is provisioned immediately the state will be active and this might fail. 
Unless we need to verify this state, we can directly poll for active state

Reply:
Agree, this probably doesn't need to be here. But it would also be surprising to get through provisioning almost instantaneously.

Reply:
sure.